test/bpf/verifier-test.sh

```diff
@@ -157,31 +152,6 @@ function cg_prog_type_init {
 	fi
 }
 
-function load_sockops_prog {
```

Comment:
This is still called in `load_cg`. We should also remove the related `mkdir` there.